.github/workflows/icu_envtest.yml

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Action configuration script for ICU environment test tasks.
+
+name: GHA EnvTest
+
+on:
+  schedule:
+    # this cron schedule is set to run 10:23 UTC every SAT
+    - cron: '23 10 * * SAT'
+
+  # For BRS:
+  # Uncomment the following two lines in a commit and push that to a PR on github
+  # will trigger the testing in addition to the schedule weekly run above.
+  pull_request:
+    branches: '**'
```

Comment:
Please ask @mihnita for instructions for setting up a command line interface for a manual trigger.

Reply:
@mihnita Is workflow_dispatch stated in  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch what you are talking about?

Reply:
it is hard to debug how to set up that manual trigger without this yaml file in the repo. Could we resolve that part later? I put in workflow_dispatch but I am not 100% sure it work unless we land this. (not sure would that only show up after the config is landing or not)

Reply:
Yes, it is `workflow_dispatch`

> it is hard to debug how to set up that manual trigger without this yaml file in the repo

I was able to play with it in a fork + branch. I can help resolve it later, or in this PR, your call.

Reply:
FYI, after configuring `workflow_dispatch`, you are not limited to the CLI for a manual trigger. You can also use the UI, which seems much less cumbersome and error-prone: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui